src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -68,6 +70,7 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
 
         try {
             storage.saveNusave(model.getNusave());
+
```

Comment:
i think we should leave these empty lines out, i.e. shouldn't have these empty lines!